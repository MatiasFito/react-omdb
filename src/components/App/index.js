import React from 'react';
import Typography from '@material-ui/core/Typography';

const App = () => {
  return (
    <div>
      <Typography variant="h2">
        Hello World
      </Typography>
      <Typography variant="subtitle1">
        Welcome to OMDB
      </Typography>
    </div>
  );
};

export default App;
