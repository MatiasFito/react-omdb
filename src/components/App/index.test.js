import React from 'react';
import shallow from 'enzyme/shallow';
import App from './index';

describe('App test suit', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<App />);
  })

  it('should render App component', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
})
