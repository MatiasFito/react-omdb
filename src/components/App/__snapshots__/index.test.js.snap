// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test suit should render App component 1`] = `
"<div>
  <WithStyles(ForwardRef(Typography)) variant=\\"h2\\">
    Hello World
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography)) variant=\\"subtitle1\\">
    Welcome to OMDB
  </WithStyles(ForwardRef(Typography))>
</div>"
`;
